go-controller/pkg/ovn/policy.go

```diff
@@ -509,7 +509,12 @@ func deleteMulticastAllowPolicy(nbClient libovsdbclient.Client, ns string, nsInf
 	ingressMatch := getACLMatch(portGroupName, getMulticastACLIgrMatch(nsInfo), knet.PolicyTypeIngress)
 	ingressACL := buildACL(ns, portGroupName, "MulticastAllowIngress", nbdb.ACLDirectionToLport, types.DefaultMcastAllowPriority, ingressMatch, nbdb.ACLActionAllow, "", knet.PolicyTypeIngress)
 
-	ops, err := libovsdbops.DeleteACLsFromPortGroupOps(nbClient, nil, portGroupName, egressACL, ingressACL)
+	matchAcls, err := libovsdbops.GetACLs(nbClient, egressACL, ingressACL)
+	if err != nil {
+		return err
+	}
+
+	ops, err := libovsdbops.DeleteACLsFromPortGroupOps(nbClient, nil, portGroupName, matchAcls...)
```

Comment:
So as per the discussion we had in this PR with @npinaeva , we don't actually need to call `DeleteACLsFromPortGroupOps` here.

The port group is deleted below, and since the ACLs are strongly referenced from them, they would be removed through garbage collection at the server.
See `isRoot documentation in https://datatracker.ietf.org/doc/html/rfc7047.txt#section-3.2